config/software/libedit.rb

```diff
@@ -24,6 +24,9 @@
 dependency "ncurses"
 dependency "config_guess"
 
+# version_list: url=http://thrysoee.dk/editline/ filter=*.tar.gz
+
+version("20210419-3.1") { source md5: "e288b8fe53bf0d096fb13847d4e0f640" }
```

Comment:
Can we change these SHAs to sha256, and update the default version to the latest version?

Reply:
Sure